### DISCUSSION TITLE
Adiitional fields for Users module

### DISCUSSION THREAD

**Discussion by vovpff** [YetiForceCompany/YetiForceCRM#14316] [How to]
I want to add 2 additional field for Users module: First name and Last name for using inside my country. Like name in English and in Russian.

What is proper way to implement them?

**Comment by mariuszkrzaczkowski** [comment]
What about the employees module?

**Reply by vovpff** [reply]
@mariuszkrzaczkowski I don't want to use employees because all employees is system users.

**Reply by mariuszkrzaczkowski** [reply]
this module was created only for such purposes, you can automate the fact that it creates a worker when creating a user

**Reply by vovpff** [reply]
@mariuszkrzaczkowski I will check it but at first look this is more difficult than add 2 text fields.

**Reply by mariuszkrzaczkowski** [reply]
user module has a different role in the system, although adding fields would probably work as well, so your decision
